extension/controller/translation/translationWorker.js

```diff
@@ -573,7 +573,7 @@ class TranslationHelper {
                 vectorResponseOptions.push_back({
                     qualityScores: false,
                     alignment: true,
-                    html: false,
+                    html: true,
```

Comment:
@andrenatal We should set this flag programmatically. It should be passed from the files in `view` (based on whether the text to be translated is html or plain) to the mediator and then should reach here so that the responseOptions can be constructed per item of the batch.